lib/primitives/object-element.js

```diff
@@ -23,6 +23,29 @@ module.exports = function(BaseElement, ArrayElement, MemberElement) {
       }, {});
     },
 
+    toEmbeddedRefract: function() {
+      if (this.attributes.content.length === 0 && this.meta.content.length === 0) {
+        var refract = {};
+
+        this.forEach(function(value, key) {
+          refract[key.toValue()] = value.toEmbeddedRefract();
+        });
+
+        return refract;
+      }
+
+      return {
+        refract: this.toRefract()
+      };
+    },
+
+    fromEmbeddedRefract: function(embeddedObj) {
```

Comment:
Looks the same as: https://github.com/refractproject/minim/pull/56/files#diff-223850cdafb14bcee834544fd3fc54e2R126

Am I wrong?